### PR TITLE
Improve Sort parsing - accept single hash / string

### PR DESCRIPTION
**Pull request by vnegrisolo**
- [x] single hash
- [x] array of hashes
- [x] single string
- [x] array of strings

### CODE REVIEW

**Comment by rubensmabueno** [comment]
:+1:

File: lib/elasticband/sort.rb [hunk 1]
```diff
@@ -9,32 +9,51 @@ class << self
...
+    private
+
+    def parse_param(param)
+      return param if param.is_a?(Hash)
+
+      { field(param).to_sym => direction(param) }
+    end
+
+    def field(param)
+      param.start_with?('-') || param.start_with?('+') ? param[1..-1] : param
```
Comment (brennovich) [inline lib/elasticband/sort.rb]: Can we freeze all these `'-'` `'+'` `','` strings?

Comment (vnegrisolo) [inline lib/elasticband/sort.rb]: good catch

**Comment by vnegrisolo** [comment]
done @brennovich ... @rubensmabueno could you approve it again please? I just added some `freeze` method calls

**Comment by brennovich** [comment]
Thank you @vnegrisolo sorry to bother you with that ;) But I believe that as an opensource library this should impact as minimum as possible in apps that are using it

:+1:

**Comment by vnegrisolo** [comment]
@brennovich this is not a bother thing... I agree with you about libraries.. the small the best

**Comment by ggcampinho** [comment]
:+1:

**Pull request merged**